sdk/cosmos/azure-cosmos/CHANGELOG.md

```diff
@@ -3,6 +3,7 @@
 ### 4.3.2 (Unreleased)
 
 #### Features Added
+- Added **preview** partial document update (Patch API) functionality and container methods for patching items with operations. For more information on Patch, please see [Azure Cosmos DB Partial Document Update](https://learn.microsoft.com/azure/cosmos-db/partial-document-update).
```

Comment:
Assuming this is a preview feature, do you want to change your version number to 4.4.0b1?